auth/services/irma/validator_test.go

```diff
@@ -0,0 +1,617 @@
+/*
+* Nuts auth
+* Copyright (C) 2021 Nuts community
+*
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package irma
+
+import (
+	irma "github.com/privacybydesign/irmago"
+	irmaservercore "github.com/privacybydesign/irmago/server"
+)
+
+//import (
```

Comment:
lots of comments here, state?

Reply:
added TODO and link to issue (https://github.com/nuts-foundation/nuts-node/issues/84)